tests/test.sh

```diff
@@ -13,7 +13,7 @@ fi
 npm run lint
 
 # Run unit tests
-NODE_ENV=testing ./node_modules/.bin/jest ./**/*.spec.js
+NODE_ENV=testing ./node_modules/.bin/jest ./**/*.spec.js && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
```

Comment:
The problem with this is that we'll always be sending coverage which isn't a good thing.

What we did before was to add a check in the shell script which was only added in the CI command for running test. Check [this out](https://github.com/BolajiOlajide/ng-faker/blob/master/tests/test.sh#L28)

and the command for testing then which is [this](https://github.com/BolajiOlajide/ng-faker/blob/master/.circleci/config.yml#L37)

Reply:
Now that I think of it, this ☝️ might not be necessary. Why do we need to run the coverage command here since we're already doing it in the CI config already.

Meaning this isn't needed.

Reply:
Did we agree not to send coverage if test fails? If we did I think this approach won't be necessary, I will just remove `&& cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js`.

Reply:
@BolajiOlajide ☝️